arm/apimanagement/apioperations.go

```diff
@@ -47,20 +45,20 @@ func NewAPIOperationsClientWithBaseURI(baseURI string, subscriptionID string) AP
 // CreateOrUpdate creates a new API operation or updates an existing one.
 //
 // resourceGroupName is the name of the resource group. serviceName is the name
-// of the API Management service. apiID is aPI identifier. Must be unique in
+// of the API Management service. aPIID is aPI identifier. Must be unique in
```

Comment:
Well, I actually prefer `apiID` but I'm not sure if there's anyway to get that back without causing further breaking changes.